configure.ac

```diff
@@ -366,6 +368,11 @@ else
 	CPPUTEST_CPPFLAGS="${CPPUTEST_CPPFLAGS} -include ${srcdir}/include/CppUTest/MemoryLeakDetectorMallocMacros.h"
 fi
 
+# Disablong long long support
+if test "x${use_longlong}" = xno; then
+  AC_DEFINE([DISABLE_LONG_LONG], 1, [enable long long])
+fi
```

Comment:
`LONG_LONG_DISABLED` might be a more consistent name with some of the other options.  Though it doesn't really matter.  Also the comment text is a little misleading.